### ISSUE TITLE
Execution option eval: false vs "Run above"

### ISSUE THREAD

**Issue by werkstattcodes** [quarto-dev/quarto-cli#2719]
`eval: false` behaves differently in RStudio and VSCode. 

When triggering "Run Above",  preceding chunks with `eval:false` are nevertheless executed when using VSCode. 
In RStudio, however, it works as intended (chunk is not executed). There is a work around (see discussion), 
but for consistency reasons it might be better to standardize the behavior.

### Discussed in https://github.com/quarto-dev/quarto-cli/discussions/2643

<div type='discussions-op-text'>

<sup>Originally posted by **werkstattcodes** September 28, 2022</sup>
I am struggling with the following, rather basic thing:

Setting the chunk option to eval: "false" makes quarto not evaluate the chunk when rendering the document. Only the code inside the chunk is displayed. 

When working 'interactively' (?), i.e. not rendering, on a quarto document, I frequently use the command "Run Above" or "Run Cells Above". In these cases, it seems like the chunk option "eval: false" is not respected, i.e. the code in the chunk in question is nevertheless evaluated. This is particularly annoying when the code in the chunk is very time intensive. As a work-around, I always have to comment out these chunks.

In short, my question, is there a way to get chunks not evaluated when not rendering the quarto document, but simply working on it?
Many thanks.

</div>
